src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -507,6 +507,7 @@ def update_progress(cur_loss, iteration):
         'training_iterations': training_iterations,
         'max_iterations': max_iterations,
         'training_loss': progress['smoothed_loss'],
+        'use_tensorflow': params['use_tensorflow']
```

Comment:
Please do not add this to the state here. We are getting rid of this codepath anyway